source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift

```diff
@@ -79,12 +88,26 @@ class ACRChoiceButton: NSView, NSTextFieldDelegate, InputHandlingViewProtocol {
     }
     
     override func mouseEntered(with event: NSEvent) {
-        guard let contentView = event.trackingArea?.owner as? ACRChoiceButton else { return }
-        contentView.button.isHighlighted = true
+        button.isHighlighted = true
     }
     override func mouseExited(with event: NSEvent) {
-        guard let contentView = event.trackingArea?.owner as? ACRChoiceButton else { return }
-        contentView.button.isHighlighted = false
+        button.isHighlighted = false
+    }
+    
+    private func setupButtonImage() {
```

Comment:
```suggestion
    
    private func updateButtonImage() {
```
Maybe `update` makes more sense here?